client/src/legacy/AddToCampaignForm.js

```diff
@@ -0,0 +1,108 @@
+/* global window */
```

Comment:
This is JS that used to be in admin https://github.com/silverstripe/silverstripe-admin/pull/1857